posts/_markdown/2022_dynamical_decoupling_in_mitiq.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Adding Digital Dynamical Decoupling to Mitiq
+date: 21 June 2022
+author: UF Team
+---
+
+
+The Unitary Fund team is excited to announce the addition of dynamical decoupling features to Mitiq, the open-source quantum toolbox in Python that mitigate errors of NISQ devices.
+
+
+Dynamical decoupling is a technique originally developed at the pulse level and in order to make it straightforward to apply also with most gate-based quantum computers and simulators, the Mitiq community has designed a **digital dynamical decoupling** (DDD) module for Mitiq, `mitiq.ddd`.
```

Comment:
```suggestion
Dynamical decoupling is a technique originally developed at the pulse level and has been modified to work in the context of gate-based quantum computers in **digital dynamical decoupling** (DDD).
The Mitiq community has designed and implemented this technique into it's own module `mitiq.ddd`.
```

This sentence felt a bit like a run-on. I suggested a change, but feel free to modify it in another way, or leave it!